nephio-5gc-controller/apis/nf/v1alpha1/upf_types.go

```diff
@@ -20,66 +20,66 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type UpfN3 struct {
+type UPFN3 struct {
 	Endpoints []Endpoint `json:"endpoints"`
 }
 
-type UpfN4 struct {
+type UPFN4 struct {
 	Endpoints []Endpoint `json:"endpoints"`
 }
 
 type N6Endpoint struct {
-	Dnn         string   `json:"dnn"`
-	IpEndpoints Endpoint `json:"ipendpoints"`
+	DNN         string   `json:"dnn"`
+	IPEndpoints Endpoint `json:"ipEndpoints"`
```

Comment:
@s3wong why is this plural when it's not a slice?

Reply:
The number of IP addresses can be more than one. But the model for this is that for each DNN, it should only have one pool and one set of IP addresses for the same gateway (i.e., presumably the same IP subnet)

But yes, removing the "s" would be less confusing.

Reply:
Ok, I pushed a new commit with the "s" removed. But I'm still a little confused on exactly what we're modeling here.